assets/images/check.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We could use react components here instead of SVGs. I would be interested in hearing Kofi's thoughts on this, and if it would be worth doing.